tech-far-hub/src/components/navigation.tsx

```diff
@@ -173,7 +173,14 @@ const Navigation = ({ isNavExpanded, onNavExpanded }: iNavigation) => {
         mobileExpanded={isNavExpanded}
         onToggleMobileNav={onNavExpanded}
       >
-        <Search onSubmit={() => false} />
+        <Search
+          onSubmit={(event) => {
+            event.preventDefault();
+            const baseUrl = `https://search.usa.gov/search?affiliate=techfar-hub&commit=Search&query=`;
+            const query = encodeURIComponent(window.document.getElementById("search-field")?.value);
```

Comment:
Can you use the `event.target.value` rather than `window.document.getElementById()`?

Reply:
the target is the search button when you click it. also i am an old timer.